frontend/src/components/frequentlyUsedFunctionsAndVariables.js

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
A file név epic, mosolyt csalt az arcomra így reggel :D Utils-nak szokták hívni általában az ilyen file-okat, de én azt sem szeretem, mert ide be lehet dobálni bármit is. Jobb, ha funkció szerint szét van szedve külön file-okra, mondjuk, textHelpers vagy constants. A másik, hogy ez nem React komponens, lehet nem is ebben a mappában kellene lennie.

Reply:
Kivittem /src-be és átneveztem!